Representable/representable
# Homepage map split

**changes**
- Adds the entry preview page which all the "Add my community" links now point to.
- Splits off the map from the index page into its own page after the entry preview page

**to test**
- click any of the "Add my community" buttons or links and you should be redirected to the entry preview page
- click on the "Add my community" button on the entry preview page and you should be taken to the entry state selection page

**screenshots**
*Desktop*
![image](https://user-images.githubusercontent.com/28410610/100191024-46aa1d00-2ea4-11eb-8896-7956408c008d.png)

*Ipad*
![image](https://user-images.githubusercontent.com/28410610/100191120-76592500-2ea4-11eb-889b-22419b1e2f36.png)

*Mobile*
![image](https://user-images.githubusercontent.com/28410610/100190965-28dcb800-2ea4-11eb-9d22-497f48c67a96.png)

*Entry State Selection Page*
![image](https://user-images.githubusercontent.com/28410610/100191413-e9fb3200-2ea4-11eb-8960-b0ac4601e163.png)
